### PR TITLE
[Fix #56] Fixed statistics mobile view for some resolution

### LINKED ISSUE #56: Speakers, 4 days event section gets left aligned at certain resolutions
issue

### PR DESCRIPTION
**Pull request by pradeepgangwar**
Issue No. - #56 
Link to view Changes - [Gh-Pages](http://pradeepgangwar.tech/inpycon2017/)

**Changes Made:**
- Added **media CSS** query for particular resolution as bootstrap grid didn't solved the problem even with full implementation.

### CODE REVIEW

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Can you please use default bootstrap classes?

Also, please avoid inline styling.

**Comment by pradeepgangwar** [comment]
@CuriousLearner In my **merged PR** #30 . I Implemented changes for mobile view using only bootstrap for all resolutions possible. But #56 issue was raised. Actually problem was only between **resolution 480px to 569px**, so I decided to change it using media queries which will not affect other classes on website. 
I removed inline styles.

**Comment by CuriousLearner** [comment]
Breakpoints eventually cause trouble when the code base size increases. Can you please use bootstrap grid instead?

**Comment by pradeepgangwar** [comment]
@CuriousLearner I have fixed it now with **bootstrap grid** only. The problem was caused by a supportive CSS file that had such rules for that resolution. The issue looks fixed now.

File: css/main.css [hunk 1]
```diff
@@ -4925,3 +4925,7 @@ body.tg-comingsoonpage .tg-foorterbar .tg-socialicons{padding:0;}
 	text-shadow:none;
 }
 .tg-alertcontent .tg-btn{float:right;}
+
+.hr-statistics{
+	color: white;
+}
```
Comment (CuriousLearner) [inline css/main.css]: Leave a line at the end of the file.

Comment (pradeepgangwar) [reply css/main.css]: @CuriousLearner  done. But I don't know why it's not showing here.

Comment (realslimshanky) [reply css/main.css]: @pradeepgangwar your new commit has been added to this PR when you pushed. The review you are seeing is showing the code at the moment of review and it'll remain same.

Comment (pradeepgangwar) [reply css/main.css]: @realslimshanky fine. @CuriousLearner please review the PR with new changes.

**Comment by jasonbraganza** [comment]
Can be closed now

**Comment by pradeepgangwar** [comment]
@CuriousLearner Please see this. I think it can be merged now.

**Review by sayanchowdhury — APPROVED** [review]

**Comment by sayanchowdhury** [comment]
LGTM :chocolate_bar:

**Pull request merged**